### ISSUE TITLE
nvda 2017.4 and google translate website with latest google chrome

### ISSUE THREAD

**Issue by joshknnd1982** [nvaccess/nvda#7845]
to reproduce this issue.
1. use latest NVDA 2017.4 and the latest google chrome browser.
2. go to the website: translate.google.com 
3. press insert space to go into focus mode.
4. press tab to the collapsed list view, its right after the detect language button. press enter to expand the list view.
5. try to press down arrow and choose another language. you can't do that because rather than choosing other radio buttons in the now expanded list box, NVDA puts you back into browse mode and the list is collapsed once again. 

6. Now try the same thing steps one through five, but in firefox 52 ESR version. This time it works. NVDA lets you select radio buttons in the expanded list view in firefox, but it does not work in chrome. 
Please fix NVDA browse mode in chrome so it works good with these expandable list boxes in google chrome so I can more easily use google translate without switching back to firefox. thanks.

**Comment by ehollig** [comment]
Using Chrome Version 63.0.3239.108, I am not able to reproduce this issue. I am able to select the options in the selection box in focus mode using NVDA version: master-14729,3152eacc.
I do, however, get a lot of verbose speech, copied from speech viewer.
list  collapsed  subMenu
list item  not selected  1 of 104
Detect language  radio menu item  checked  1 of 1
Google Translate  document
list  expanded  subMenu
Google Translate  document
list  expanded  subMenu
Afrikaans  radio menu item  1 of 1
Google Translate  document
list  expanded  subMenu
Google Translate  document
list  expanded  subMenu
Albanian  radio menu item  1 of 1

**Comment by Adriani90** [comment]
The design of google translate changed. There is no list anymore, now there are buttons which work whell in focus and browse mode. Testing with NVDA 2019.1.1 and Chrome 74.
Closing as works for me.
